superset-frontend/src/pages/Home/Home.test.tsx

```diff
@@ -228,3 +228,26 @@ test('Should render a submenu extension component if one is supplied', async ()
 
   expect(screen.getByText('submenu extension')).toBeInTheDocument();
 });
+
+test('Should not make data fetch calls if `welcome.main.replacement` is defined', async () => {
+  const extensionsRegistry = getExtensionsRegistry();
+
+  // Clean up
+  extensionsRegistry.set('welcome.banner', () => null);
+  extensionsRegistry.set('welcome.main.replacement', () => (
+    <>welcome.main.replacement extension component</>
+  ));
```

Comment:
Minor nit: this is not a cleanup, this is part of the setup.

Reply:
that's true. I missed a blank line there

Reply:
Just addressed in a new commit